apps/portal/js/designer.js

```diff
@@ -694,21 +694,26 @@ $(function () {
                 id = componentBox.find('.ues-component').attr('id'),
                 removeBlock = true;
 
-            if (id) {
-                removeComponent(findComponent(id), function (err) {
-                    if (err) {
-                        removeBlock = false;
-                        console.error(err);
-                    }
-                    saveDashboard();
-                });
-            }
+            var removeGadgetBlock = function () {
+                if (id) {
+                    removeComponent(findComponent(id), function (err) {
+                        if (err) {
+                            removeBlock = false;
+                            console.error(err);
```

Comment:
Avoid using console.error, it could be used for debugging but not good to have it in the code.